src/app/[lng]/(common)/(needSidebar)/[category]/CategorizedNotices.tsx

```diff
@@ -19,12 +19,23 @@ const CategorizedNotices = async ({
   lng,
   page,
 }: PropsWithLng<CategorizedNoticesProps>) => {
+  const ITEMS_PER_PAGE = 10;
+
   const { t } = await createTranslation(lng);
 
   const notices = await getAllNotices(
     sortByDeadline
-      ? { ...noticeSearchParams, orderBy: 'deadline' }
-      : noticeSearchParams,
+      ? {
+          ...noticeSearchParams,
+          orderBy: 'deadline',
+          offset: page * ITEMS_PER_PAGE,
+          limit: ITEMS_PER_PAGE,
+        }
+      : {
+          ...noticeSearchParams,
+          offset: page * ITEMS_PER_PAGE,
+          limit: ITEMS_PER_PAGE,
+        },
```

Comment:
```suggestion
      {
          ...noticeSearchParams,
          offset: page * ITEMS_PER_PAGE,
          limit: ITEMS_PER_PAGE,
          ...(sortByDeadline ? {orderBy: 'deadline'} : {}),
        }
```

더 편한 방식으로 쓰면 될 것 같아용